test-requirements.txt

```diff
@@ -1,5 +1,3 @@
-black==23.3.0
-flake8
 mock
 pexpect
```

Comment:
Can you check where `pexpect` is needed in XR? We can probably remove this as well.